### DISCUSSION TITLE
This is how my MiniDexed landed…

### DISCUSSION THREAD

**Discussion by Doug2233** [probonopd/MiniDexed#320] [Show and tell]
![946586E0-5B19-4F98-BADF-6D3BD4515A6A](https://user-images.githubusercontent.com/80731201/180372141-b82de721-1386-46bb-a079-f703f3ec9c71.jpeg)

I managed to squeeze this into this box. The most difficult part was acquiring a Pi4.

![125FFA34-AC27-46BF-8506-E44BEEB569AE](https://user-images.githubusercontent.com/80731201/180372172-3f587fd7-4f06-446f-bfbb-a3ead089c2da.jpeg)

I have to say I am over the moon with this project.  I have never played an actual DX7 but there is no denying the footprint of this product is cast over at least two decades of popular music.  To be able to recreate this as a ‘rack of 8’ is truely extraordinary. Thank you to all the contributors - keep up the good work,  I hope I can contribute in some small way too.

I invited a good friend over who, apart from being an active musician has a particular interest in creating SoundFonts for a variety of instruments - he too was blown away. The audio quality and accessibility of the patches recreate the sounds so faithfully.

Doug

**Comment by sou7611** [comment]
Fascinating!  What box is that which you used?  Looks sleek.

I never found a Pi4, but I found a Pi3a, which tend to run a bit cheaper
'cause they don't have onboard ethernet, and only one USB.  Neither of
which affects the Minidexed project at all.

**Reply by Doug2233** [reply]
This is the box: https://www.jaycar.com.au/jiffy-box-black-158-x-95-x-53mm/p/HB6011?pos=1&queryId=4460fde0da6520be074aef67399b37d0
A pretty common electronics project box in this part of the world.  Origin is China so try the common market places.
The Pi4 seems overkill but the P4, Pi Zero and Pi Zero2W are the only versions on my radar.  I will switch this project to a Zero2W when I can.  I only have this Pi4 since it’s original intended use was accounted for with a Zero2W.
